### PR TITLE
Create page-speed.md

### PR DESCRIPTION
**Pull request by CAG07**
A summary of page speed with an example.

### CODE REVIEW

File: page-speed.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<h1>Page Speed</h1>
```
Comment (PatricNox) [inline page-speed.md]: use ## for H1

and ### for H2

Comment (PatricNox) [reply page-speed.md]: Also, put this in https://github.com/fossbarrow/ultimate-seo-checklist/tree/main/All-About-SEO directory

Comment (CAG07) [reply page-speed.md]: I made the requested changes, but am having a terrible time trying to change the source branch for the PR to the path you requested. Should I create a new PR?

Comment (PatricNox) [reply page-speed.md]: Should just be to move this file into that directory and then commit it :)

Comment (CAG07) [reply page-speed.md]: I uploaded the file under the path /All-About-SEO and committed. Thank you for your patience.

File: page-speed.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+## Page Speed
```
Comment (PatricNox) [inline page-speed.md]: You can remove this file now! 

/page-speed.md

Comment (CAG07) [reply page-speed.md]: This has been removed.

File: page-speed.md (hunk 1 above)
Comment (CAG07) [reply page-speed.md]: @PatricNox Can you please move this over to /All-About-SEO? I'm not seeing where I can move this when comparing to mine.

**Comment by PatricNox** [comment]
Thank you @CAG07 for your dedication, great addition! :)

**Comment by CAG07** [comment]
@PatricNox Thank you again for your patience and I look forward to contributing more in the future!

**Pull request merged**